oppijanumerorekisteri-service/src/test/java/fi/vm/sade/oppijanumerorekisteri/controllers/HenkiloControllerTest.java

```diff
@@ -134,7 +131,7 @@ public void getHenkiloYhteystiedot() throws Exception {
                 "  }" +
                 "}";
         given(this.henkiloService.getHenkiloYhteystiedot("1.2.3.4.5")).willReturn(new HenkilonYhteystiedotViewDto()
-            .put(YhteystietoRyhmaKuvaus.KOTIOSOITE, YhteystiedotDto.builder().sahkoposti("testi@test.com").build()));
+            .put("kotiosoite", YhteystiedotDto.builder().sahkoposti("testi@test.com").build()));
```

Comment:
Oikea ajonaikaisesti käytettävä arvo kotiosoitteelle olisi "yhteystietotyyppi1". Pitäisikö näitä käyttää myös testeissä?